src/QsCompiler/DataStructures/ReservedKeywords.fs

```diff
@@ -198,6 +198,9 @@ module InternalUse =
     /// to be used as name for all namespaces that do not have a valid name
     let UnknownNamespace = "__UnknownNamespaceName__"
 
+    /// the implicit namespace in notebooks
+    let NotebookNamespace = "__NotebookNamespace__"
```

Comment:
Xinyi raised a very good point that maybe each notebook should have its own namespace. However, AzureNotebooks currently spawns a new language server for each notebook, so I imagine it should be okay to stick to this for now

Reply:
I can change it to UnknownNamespace if that's better. I know you'd suggested to use UnknownNamespace but I guess that felt off to me, since the implicit notebook namespace isn't _unknown_ per se, right? It's more of an implicit namespace? However, I understand a placeholder is a placeholder.

By the way: As I was testing (struggling) with semantic stuff, I noticed that autocomplete suggestions included `__NotebookNamespace__`. Do you think we should filter that (or `__UnknownNamespaceName__`) out?